assets/dev/scss/frontend/widgets/button.scss

```diff
@@ -88,11 +88,6 @@
 	}
 }
 
-/* TODO: REMOVE THIS EXCEPTION AFTER PRO 2.8 RELEASES */
```

Comment:
Check

Reply:
Is needed, I just removed the TODO comment. This code needs to stay permanent, or get moved to a different file.

Reply:
Return the comment back